src/CalendarViews/MonthlyCalendar/MainBody/StartOfMonthBuffer/StartOfMonthBuffer.tsx

```diff
@@ -0,0 +1,21 @@
+interface Props {
+    firstOfMonth: number
+}
+
+/**
+ * Creates a buffer div for the start of the month
+ * @param props The information needed to be passed in
+ * @returns If the first day is a sunday, an empty element. 
+ *          Otherwise, a buffer element between sunday and the first date.
+ */
+function StartOfMonthBuffer(props: Props){
+    let { firstOfMonth } = props
```

Comment:
```suggestion
function StartOfMonthBuffer({ firstOfMonth }: Props){
```

I would recommend destructuring like this, keeps it all in one line all neatly.